src/evidently/analyzers/stattests/registry.py

```diff
@@ -0,0 +1,37 @@
+from typing import List, Union, Callable, Tuple, Dict
+
+REGISTERED_STAT_TEST: Dict[str, Dict[str, Callable]] = {}
+
+
+def stattest(name: str, allowed_feature_types: List[str]):
+    def wrapper(func):
+        REGISTERED_STAT_TEST[name] = {feature_type: func for feature_type in allowed_feature_types}
+        return func
+
+    return wrapper
+
+
+def get_stattest(stattest_func: Union[str, Callable], feature_type: str) -> Tuple[str, Callable]:
```

Comment:
Maybe use `stattest` in packages/modules naming, and in functions use one option - `stat_test` ?